CMakeLists.txt

```diff
@@ -362,6 +362,14 @@ if(WITH_VTK AND NOT ANDROID)
   find_package(VTK)
   if(VTK_FOUND)
     message(STATUS "VTK_MAJOR_VERSION ${VTK_MAJOR_VERSION}")
+    if (NOT "${VTK_RENDERING_BACKEND}" STREQUAL "OpenGL")
```

Comment:
VTK! It is set by the VTK CMake script in newer VTK versions:

```
$ sudo grep -i "VTK_RENDERING_BACKEND" /opt/vtk7_opengl2/lib/cmake/vtk-7.1 -r
/opt/vtk7_opengl2/lib/cmake/vtk-7.1/VTKConfig.cmake:set(VTK_RENDERING_BACKEND "OpenGL2")
/opt/vtk7_opengl2/lib/cmake/vtk-7.1/vtkGroups.cmake:    if(${group} MATCHES "^Rendering" AND "None" STREQUAL VTK_RENDERING_BACKEND)
```

This variable does not exist in VTK 5.8 release so I changed the condition to handle that case.
Still, VTK is not disabled when OpenGL2 is detected (I get the warning message though).